backend/src/main/resources/db/migration/common/V1_0_15__Add_Esal_Register_Fields.sql

```diff
@@ -0,0 +1,18 @@
+ALTER TABLE esals
+    ADD COLUMN logo_url VARCHAR(255);
+ALTER TABLE esals
+    ADD COLUMN webpage VARCHAR(255);
+ALTER TABLE esals
+    ADD COLUMN description VARCHAR(500);
```

Comment:
This should be validated at request level too. It is better to reply with `400 description too long` instead of `500 could not register esal`

Reply:
Ok, i added @_Size in both fields in ESALRequestDto